takeshape/penny
# Cart components

### Test Plan (Steps to test):

1. Check out the stories
1. Go to the Vercel site and:
* Add to cart
* Remove from cart
* Checkout when not signed in
* Checkout when signed in
* Increase cart quantity
* Decrease cart quantity
* Add an item as both recurring and one-time
* Add an item already in the cart to the cart (should increase quantity)
